tests/lib/Repository/User/PasswordHashServiceTest.php

```diff
@@ -40,6 +41,45 @@ public function testIsHashTypeSupported(): void
         $this->assertTrue($this->passwordHashService->isHashTypeSupported(User::DEFAULT_PASSWORD_HASH));
         $this->assertFalse($this->passwordHashService->isHashTypeSupported(self::NON_EXISTING_PASSWORD_HASH));
     }
+
+    public function testCreatePasswordHashThrowsUnsupportedPasswordHashType(): void
+    {
+        $this->expectException(UnsupportedPasswordHashType::class);
+
+        $this->passwordHashService->createPasswordHash('secret', self::NON_EXISTING_PASSWORD_HASH);
+    }
+
+    public function testCreatePasswordHashExceptionHidesSensitiveParameter(): void
+    {
+        $ignoreArgs = ini_get('zend.exception_ignore_args');
+        $paramMax = ini_get('zend.exception_string_param_max_len');
+
+        ini_set('zend.exception_ignore_args', 0);
+        ini_set('zend.exception_string_param_max_len', 10);
+
+        $password = 'secret';
+
+        try {
+            $this->passwordHashService->createPasswordHash($password, self::NON_EXISTING_PASSWORD_HASH);
+        } catch (UnsupportedPasswordHashType $e) {
+            $stackTrace = $e->getTrace();
+            $this->assertIsArray($stackTrace);
+            $this->assertGreaterThan(1, count($stackTrace));
+            $this->assertArrayHasKey('function', $stackTrace[0]);
+            $this->assertEquals('createPasswordHash', $stackTrace[0]['function']);
+            $this->assertArrayHasKey('args', $stackTrace[0]);
```

Comment:
Static methods should be called statically
```suggestion
            static::assertIsArray($stackTrace);
            static::assertGreaterThan(1, count($stackTrace));
            static::assertArrayHasKey('function', $stackTrace[0]);
            static::assertEquals('createPasswordHash', $stackTrace[0]['function']);
            static::assertArrayHasKey('args', $stackTrace[0]);
```

Reply:
Hey @glye when I wrote this suggestion, my brain must've stopped working. While it's a call to a static method, we always use `self::`. I pushed a commit fixing that. Sorry for the fuss.